docs/user_guide.md

```diff
@@ -163,6 +163,36 @@ Content-Length: 537
 }
 ```
 
+#### Job execution API
+
+Jobs can be executed by incoming webhook.
+Incoming webhook uses basic authentication and authentication information can be set at `api_basic_authentication_applications` key in kuroko2.yml.
```

Comment:
How about change to below sentence ?

The execution API requires basic authentication. Set the `api_basic_authentication_applications` key in kuroko2.yml if you enable the API.